src/components/AppSettingsMenu.vue

```diff
@@ -267,9 +267,16 @@ p.app-settings {
 		line-height: normal;
 		min-height: 44px;
 		font-size: unset;
+		opacity: .7;
```

Comment:
This also changes the opacity of the button text. Is this intended? I don't like it because it negatively affects contrast/accessibility.

[Peek 2022-07-12 15-26.webm](https://user-images.githubusercontent.com/1479486/178500936-9162335d-a3b7-4636-871a-780c1bf4d70c.webm)

Reply:
damn, no its not intended, will fix now